### PR TITLE
feat(block/audio): Blue Aercloud Bouncing Sounds

### PR DESCRIPTION
**Pull request by Zepalesque**
The sounds for bouncing on Blue Aerclouds from the Aether II.
Also includes a configuration option to disable the sounds.

---
I agree to the Contributor License Agreement (CLA).

### CODE REVIEW

**Comment by Zepalesque** [comment]
Hmm, I think I can probably make this configurable on the client, so I’ll try doing that at some point soon.

**Comment by Zepalesque** [comment]
alright, done

(of course as long as there are no issues I missed, that is)

File: src/generated/resources/assets/aether_genesis/lang/en_us.json [hunk 1]
```diff
@@ -46,5 +47,6 @@
   "item.aether_genesis.wrapped_chocolates": "Wrapped Chocolates",
   "item.aether_genesis.wyndberry": "Wyndberry",
   "menu.aether_genesis.holystone_furnace": "Holystone Furnace",
-  "pack.aether.mod.description": "Genesis Resources"
+  "pack.aether.mod.description": "Genesis Resources",
+  "subtitles.aether.block.aercloud.blue_aercloud_bounce": "Blue Aercloud Bounces"
```
Comment (bconlon1) [inline src/generated/resources/assets/aether_genesis/lang/en_us.json]: Bounces shouldn't be capitalized.

**Review by bconlon1 — APPROVED** [review]
lgtm

**Pull request merged**